### ISSUE TITLE
CI about to fail due to build-crd-testing deletion

### ISSUE THREAD

**Issue by bobcatfish** [tektoncd/pipeline#4220]
@mattmoor pointed out that the GCP project `build-crd-testing` has been deleted; we have a bunch of examples which reference this project (e.g. https://github.com/tektoncd/pipeline/blob/bda40ea059cbb54a0d6c206b756d735b72df5b74/examples/v1beta1/taskruns/pull-private-image.yaml#L50) and we also use it in an end to end test (https://github.com/tektoncd/pipeline/blob/bda40ea059cbb54a0d6c206b756d735b72df5b74/test/pipelinerun_test.go#L696).

We'll need to update these examples and tests to use a project that actually exists (e.g. one of these https://github.com/tektoncd/plumbing/blob/main/docs/README.md#gcp-projects); in the meantime it might be best to disable these if they start blocking PRs.

**Comment by vdemeester** [comment]
Agreed 😅 What was the "secret sauce" doing ? I guess only used as a private image to exercise the auth ?

**Comment by mattmoor** [comment]
Yup, private image auth test

**Comment by bobcatfish** [comment]
I'm gonna downgrade the priority on this one since @vdemeester has at least made it so this won't block folks

**Comment by sbwsg** [comment]
One of the tricky parts here is that GCR only supports ACL at the registry level, so we either need to use an existing private registry we have in a project or create a new project explicitly for testing this out.

The `tekton-nightly` project has existing private registries set up for `eu.gcr.io`, `us.gcr.io` and `asia.gcr.io`. We could use one of these? @tektoncd/core-maintainers wdyt?

**Comment by wlynch** [comment]
I'd recommend against using `tekton-nightly` out of paranoia that tests might gain unintended access to other dogfooding resources.

My preference would be to use a new project and/or use [Artifact Registry which has finer ACLs than GCR](https://cloud.google.com/artifact-registry/docs/access-control).

**Comment by sbwsg** [comment]
@afrittoli is working on running tests in Kind and with this it should be possible to setup a registry with credentials for testing purposes. From Pipelines WG just now this would be preferable over maintaining a piece of cloud infra specific to tests so that's the approach we intend to pursue before committing to a new GCP project, Artifact Registry or similar.

**Comment by vdemeester** [comment]
I think I'll go ahead and close this one.
